diktat-rules/src/test/kotlin/org/cqfn/diktat/util/DiktatRuleSetProvider4Test.kt

```diff
@@ -65,6 +63,7 @@ class DiktatRuleSetProviderTest {
         private val ignoreFile = listOf(
             "DiktatRuleSetProvider",
             "DiktatRule",
-            "IndentationError")
+            "IndentationError",
+            "OrderedRuleSet")
```

Comment:
I don’t get this logic - what is this for?

Reply:
this tests checks that all files (classes) from folder `org.cqfn.diktat.ruleset.rules` are presented in `DiktatRuleSetProvider` as rule with this exclusions

Reply:
The test collects the list of rules available at run time and compares those to the list of files in the source tree.

Here, we ignore source files which contain no rules.